src/components/controls/base/BaseSearchBar.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <input
+      :value="value"
+      @input="$emit('input', $event.target.value)"
+      id="baseSearch"
+      type="text"
+      :placeholder="placeholderText"
+    />
+    <img :src="iconSrc" />
```

Comment:
This can stay for now, but I just opened a PR to implement FontAwesome, so we can use nice icons.